config/src/main/java/org/hyperledger/besu/config/GenesisReader.java

```diff
@@ -80,7 +80,7 @@ public Stream<GenesisAccount> streamAllocations() {
                 final var on = normalizeKeys((ObjectNode) entry.getValue());
                 return new GenesisAccount(
                     Address.fromHexString(entry.getKey()),
-                    JsonUtil.getString(on, "nonce").map(ParserUtils::parseUnsignedLong).orElse(0L),
```

Comment:
would be great to add a test that verifies the desired behaviour.

Reply:
@macfarla Sure, I will begin working on it right away.